plugin/storage/es/ilm/ism-policy.json

```diff
@@ -0,0 +1,22 @@
+{
+  "policy": {
+    "description": "Jaeger default ISM Policy",
+    "default_state": "hot",
+    "states": [
+      {
+        "name": "hot",
+        "actions": [
+          {
+            "rollover": {
+              "min_index_age": "1d"
+            }
+          }
+        ]
+      }
+    ],
+    "ism_template": {
+      "index_patterns": ["*jaeger-*"],
```

Comment:
this feels too broad, it could catch unexpected indices this way. I think it should be parameterized with the index prefix of the actual store